From 8aaab68b39d5a7daa9d21b6e1481d5707973b8d6 Mon Sep 17 00:00:00 2001
From: Kondapally Kalyan <kalyan.kondapally@intel.com>
Date: Fri, 20 Sep 2013 19:08:02 +0300
Subject: [PATCH] Register messages file to IPC System.

This patch adds WaylandMsgStart to IPCMessageStart enum in ipc_message_start.h.
The XXXMsgStart value is an enumeration that ensures uniqueness for each different
message file.
---
 ipc/ipc_message_start.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/ipc/ipc_message_start.h b/ipc/ipc_message_start.h
index dc9be23..a269a15 100644
--- a/ipc/ipc_message_start.h
+++ b/ipc/ipc_message_start.h
@@ -89,6 +89,7 @@ enum IPCMessageStart {
   LocalDiscoveryMsgStart,
   PowerMonitorMsgStart,
   EncryptedMediaMsgStart,
+  WaylandMsgStart,
   LastIPCMsgStart      // Must come last.
 };
 
-- 
1.7.9.5

